src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.jsx

```diff
@@ -94,72 +93,59 @@ const SelectHareDropDown = ({
       ],
   );
 
-  const { status: statusX, data: dataX } = hareResults.at(
-    hareResults.length === 1 ? 0 : -1,
-  );
-
-  useEffect(() => {
-    setHares(dataX?.concept_breakdown);
-  }, [dataX]);
+  const getFinalPopulationSize = (selectedHareItem) => {
+    const hareBreakdownPart1 = hareResults[0].data.concept_breakdown.find((hare) => selectedHareItem.value === hare.concept_value).persons_in_cohort_with_value;
+    if (outcome.variable_type === 'concept') {
+      return hareBreakdownPart1;
+    } else {
+      const hareBreakdownPart2 = hareResults[1].data.concept_breakdown.find((hare) => selectedHareItem.value === hare.concept_value).persons_in_cohort_with_value;
+      return hareBreakdownPart1 + hareBreakdownPart2;
```

Comment:
**QUESTIONI** I pulled this local and it looks like this is working good (correctly sums control and case when there is dichotomous), otherwise it outputs the lowest value when it is non-dichotomous outcome when done without a covariate phenotype.

However, I'm not sure if it includes the attrition from the covariate  phenotype in the second table, see screenshots below for non-hispanic white. Can you confirm this is expected behavior?

<img width="1482" alt="image" src="https://user-images.githubusercontent.com/113449836/207952381-d108762d-5714-45ad-91e5-eeb6658d92d6.png">

Final Size is 99 (I thought it would be 57?)
<img width="1482" alt="image" src="https://user-images.githubusercontent.com/113449836/207952470-4758767c-6166-4482-856b-36e508001713.png">

Reply:
"other" is also a HARE group in some of our dummy data. It is not different from any of the other groups.